lti_consumer/utils.py

```diff
@@ -1,11 +1,48 @@
 # -*- coding: utf-8 -*-
 """
-Make '_' a no-op so we can scrape strings
+Utility functions for LTI Consumer block
 """
+from six import text_type
+from django.conf import settings
 
 
 def _(text):
     """
-    :return text
+    Make '_' a no-op so we can scrape strings
     """
     return text
+
+
+def get_lms_base():
+    """
+    Returns LMS base url to be used as issuer on OAuth2 flows
+
+    TODO: This needs to be improved and account for Open edX sites and
+    organizations.
+    One possible improvement is to use `contentstore.get_lms_link_for_item`
+    and strip the base domain name.
+    """
+    return settings.LMS_ROOT_URL
```

Comment:
This might not be the case for urls fetched from `SiteConfiguration`, but I don't think it works from this context.